src/components/country-datalist.ts

```diff
@@ -258,15 +260,23 @@ export const countryDisplayNames =
       })
     : undefined;
 
+export const getCountryOptions = () => {
+  const options = countries.map((country) => ({
+    value: country,
+    label: countryDisplayNames ? countryDisplayNames.of(country)! : country,
+  }));
+  options.sort((a, b) => caseInsensitiveStringCompare(a.label, b.label));
+  return options;
+};
```

Comment:
Can't we just generate the array instead of creating a function ?
Each render will call this function and sort the array.

Reply:
It needs to be translated, but we should probably memoize it